tests/data/output/enums/value/input-schema.yml

```diff
@@ -5,9 +5,11 @@ properties:
       type: string
       enum:
         - value: fincm3
+          title: fincm3
```

Comment:
the `title` is always emitted now I guess ? I think this is fine, just was curious.

Reply:
Yes.